.github/actions/vtk-install-dep/action.yml

```diff
@@ -115,6 +123,7 @@ runs:
         -DVTK_MODULE_ENABLE_VTK_IOGeometry=YES
         -DVTK_MODULE_ENABLE_VTK_IOImage=YES
         -DVTK_MODULE_ENABLE_VTK_IOImport=YES
+        -DVTK_MODULE_ENABLE_VTK_IOOpenVDB=${{ runner.os != 'macOS' && 'YES' || 'NO' }}
```

Comment:
don't you have to check for VTK version here?

Reply:
No, we dont care that the option is set without effect.